### PR TITLE
fix: some notification UI fixes

### PR DESCRIPTION
**Pull request by caybro**
This aims at bringing the notification display closer to what regular messages look like, namely:

- show compressed chat keys
- don't show identicon rings for ENS names
- fix timestamps and their tooltips (`notification` isn't part of StatusMessage!)
- unbreak clicking outside of the whole popup overlay (`activityCenter` is undefined)
- fix blurry checkmark icon for marking notification as read

### What does the PR do

Fixes some smaller UI issues around activity center notifications

### Affected areas

ActivityCenter

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

Snímek obrazovky z 2022-11-29 12-54-53

### CODE REVIEW

File: ui/StatusQ/src/StatusQ/Components/StatusMessage.qml [hunk 1]
```diff
@@ -73,8 +73,8 @@ Control {
     property StatusMessageDetails messageDetails: StatusMessageDetails {}
     property StatusMessageDetails replyDetails: StatusMessageDetails {}
 
-    property string timestampString: new Date(notification.timestamp).toLocaleTimeString(Qt.locale(), Locale.ShortFormat)
-    property string timestampTooltipString: new Date(notification.timestamp).toLocaleString()
```
Comment (PascalPrecht) [inline ui/StatusQ/src/StatusQ/Components/StatusMessage.qml:77]: Just double checking: `notification` is just magically gone?

**Review by PascalPrecht — APPROVED** [review]
Just a question but LGTM otherwise

File: ui/StatusQ/src/StatusQ/Components/StatusMessage.qml (hunk 1 above)
Comment (caybro) [reply ui/StatusQ/src/StatusQ/Components/StatusMessage.qml:77]: Yeah, this was the error, it was never supposed to be there in StatusMessage :) (some copy and paste error I suppose)

Comment (caybro) [reply ui/StatusQ/src/StatusQ/Components/StatusMessage.qml:77]: I mean, there is no such thing as a `notification` inside StatusQ

**Pull request merged**